README.md

```diff
@@ -4,47 +4,85 @@
 Placeholder for the Metadata Query Service
 
 
+# Query Service
+[![Test Coverage](https://codecov.io/gh/HumanCellAtlas/query-service/branch/master/graph/badge.svg)](https://codecov.io/gh/HumanCellAtlas/query-service)
+
+######TODO add these links
+[![Production Health Check](health check badge SVG)](URL to service)
+[![Master Deployment Status](gitlab pipeline badge SVG)](https://allspark.dev.data.humancellatlas.org/HumanCellAtlas/query-service/pipelines)
```

Comment:
Any chance of getting these badges working?

Reply:
updated master deployment status, but we dont currently have a prod env so we cant do the Prod health check yet

Reply:
https://gitlab.com/gitlab-org/gitlab-ce/issues/54633 😢

Reply:
Also note: gitlab badges are not generally available if you're not authenticated, meaning the public won't be able to view them. Use the Status API https://github.com/HumanCellAtlas/status-api